src/frontend/src/pages/login/login.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import Grid from '@material-ui/core/Grid';
+
+// for fields
+import TextField from '@material-ui/core/TextField';
+
+// for submit
+import Button from '@material-ui/core/Button';
+import {
+    Link,
+} from "react-router-dom"
 
 export const Login: React.FC = () => {
     return (
-        <div>Login Page</div>
+        <div>
```

Comment:
```suggestion
        <Container>
            <Box p={3}>
```
I would suggest using Material-UI components rather than native HTML wherever possible. I've suggested that others put everything in a ```<Container>``` element, then use a ```<Box>``` component to add padding. We may eventually abstract this away however.